.changeset/yellow-eggs-provide.md

```diff
@@ -0,0 +1,19 @@
+---
+'@twilio-paste/radio-group': minor
+'@twilio-paste/core': minor
+---
+
+[radio-group]
+
+- Add new customization element for the radio control circle
```

Comment:
nitpick (non-blocking): if you wanted to make this _even better_ you might mention what the element is, what it controls and what it's called.